spamhandling.py

```diff
@@ -84,7 +84,7 @@ def handle_spam(post, reasons, why):
     reason = ", ".join(reasons[:5])
     if len(reasons) > 5:
         reason += ", +{} more".format(len(reasons) - 5)
-    reason = reason[:1].upper() + reason[1:]  # reason is capitalised, unlike the entries of reasons list
+    reason = reason.title()  # reason is capitalised, unlike the entries of reasons list
```

Comment:
Duh.  Thanks for catching this!  Rebased and force pushed without this silly commit.